### ISSUE TITLE
Difference from @Slava's package?

### ISSUE THREAD

**Issue by dandv** [bitflak/Meteor-Xray#4]
How is this fork different from @Slava's https://github.com/Slava/Meteor-Xray/ ?

**Comment by Slava** [comment]
Mine is a fork that I merged later here

**Comment by dandv** [comment]
The problem is we have two apparently identical packages.

https://atmospherejs.com/?q=xray

**Comment by Slava** [comment]
Flag mine then

**Comment by dandv** [comment]
@Slava: yours has 10x more downloads, so my flag won't mean much. @krevativ, do you still plan to maintain this fork?

**Comment by bitflak** [comment]
@dandv yes of course I do! :) I'm writing my diploma right now, thats why nothing happens. But I'm happy to see the interest for this package, so I will try to find some time in the next few days and to fix the dev branch.
